mgmtworker/cloudify_system_workflows/search_utils.py

```diff
@@ -138,3 +169,32 @@ def get_instance_ids_by_node_ids(client, node_ids):
         else:
             break
     return ni_ids
+
+
+def operation_name_matches(operation_name, search_value,
+                           valid_values=None,
+                           operation_name_specs=None):
```

Comment:
can we rename some of those parameters, or maybe docstring them?
Because I note that the first parameter is `operation_name`, but when you call this function, the caller calls the SECOND parameter `operation_name`. That's a bit confusing :P